src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorDesignTimeNodeWriter.cs

```diff
@@ -423,7 +423,7 @@ public override void WriteComponentAttribute(CodeRenderingContext context, Compo
                 // We don't expect this to happen, we just want to know if it can.
                 throw new InvalidOperationException("Attribute nodes should either be minimized or a single content node.");
             }
-            else if (node.BoundAttribute.IsUIEventHandlerProperty())
+            else if (node.BoundAttribute?.IsUIEventHandlerProperty() ?? false)
```

Comment:
The only change to the codegen is to ignore the case where we don't have a matching property. All of the complexity here was in the lowering part.